cmd/neofs-node/config.go

```diff
@@ -969,8 +971,45 @@ func (c *cfg) configWatcher(ctx context.Context) {
 	}
 }
 
-// writeAppVersion writes app version as defined at compilation
-// step to the node's attributes.
-func writeAppVersion(c *cfg) {
+// writeSystemAttributes writes app version as defined at compilation
+// step to the node's attributes and an aggregated disk capacity
+// according to the all space on the all configured disks.
+func writeSystemAttributes(c *cfg) error {
+	// `Version` attribute
+
 	c.cfgNodeInfo.localInfo.SetAttribute("Version", misc.Version)
+
+	// `Capacity` attribute
+
+	var stat syscall.Statfs_t
+	var total uint64 // bytes
+	fsIDMap := make(map[syscall.Fsid]struct{})
+
+	for _, sh := range c.applicationConfiguration.EngineCfg.shards {
+		for _, storage := range sh.subStorages {
+			path := storage.path
+
+			err := util.MkdirAllX(path, storage.perm)
```

Comment:
I almost hate this function even though I roughly understand why it's here.

Reply:
a dir may be missing, and we will create it anyway later (or fail to create it). what exactly bothers you? `MkdirAllX` in general or using it here?

Reply:
What bothers me is that node must not create these configured directories. But it does and it relies on this behavior, that's why we have to replicate this nonsense here. #2497.